movies-manager/package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "movies-manager",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start:dev": "cross-env NODE_ENV=development webpack-dev-server",
+    "build:dev": "cross-env NODE_ENV=development webpack",
```

Comment:
build:dev command is redundant

Reply:
why, if I have 2 different bundle 1 with source map and another one with minification ?

Reply:
Because build command is only for a production (where your bundle will be minified).